views/forms/ControlledMultiSelect/index.jsx

```diff
@@ -11,10 +33,20 @@ const ControlledMultiSelect = (props) => {
         <Autocomplete
           id={props.name}
           fullWidth={props.fullWidth}
+          getOptionDisabled={(option) => option.isFixed}
           isOptionEqualToValue={(option, value) => option.value === value.value}
           multiple
-          onChange={(_, data) => field.onChange(data)}
+          onChange={(_, data, reason) => onChange(_, data, reason, field)}
```

Comment:
if we're not using the event, do we need to pass it to the `onChange` function?

```suggestion
          onChange={(_, data, reason) => onChange(data, reason, field)}
```

Reply:
[47a7352](https://github.com/AMP-SCZ/dpdash/pull/487/commits/47a7352aa90256c68d3515ee70b8a513b3052989)